packages/taxdat/tests/testthat/test_read_taxonomy_data_sql.R

```diff
@@ -0,0 +1,68 @@
+username=Sys.getenv("username")
```

Comment:
We have standard environment variables for this, don't we? They should be in set_parameters.R

Reply:
```
dbuser <- Sys.getenv("USER", "app")
dbname <- Sys.getenv("CHOLERA_COVAR_DBNAME", "cholera_covariates")
```